### PR TITLE
get courses, subjects of superviser

### PR DESCRIPTION
**Pull request by thaonp104**
## Related Tickets:
- [26059](https://edu-redmine.sun-asterisk.vn/issues/26059)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by thaonp104** [comment]
ready

File: webpack.mix.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
 const mix = require('laravel-mix');
 
-mix.js('resources/js/app.js', 'public/js')
-    .js('resources/js/createSubject.js', 'public/js')
-    .js('resources/js/trainee_calendar.js', 'public/js');
+mix.js('resources/js/createSubject.js', 'public/js/createSubject.js')
+    .js('resources/js/trainee_calendar.js', 'public/js/trainee_calendar.js')
```
Comment (hoangminhuet99) [inline webpack.mix.js:4]:
```suggestion
mix.js('resources/js/create_subject.js', 'public/js')
    .js('resources/js/trainee_calendar.js', 'public/js')
```
đặt tên file js snake case đi chị :v

**Comment by thaonp104** [comment]
ready

File: webpack.mix.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
 const mix = require('laravel-mix');
 
-mix.js('resources/js/app.js', 'public/js')
-    .js('resources/js/createSubject.js', 'public/js')
-    .js('resources/js/trainee_calendar.js', 'public/js');
+mix.js('resources/js/createSubject.js', 'public/js/createSubject.js')
```
Comment (huyenttm-0901) [inline webpack.mix.js]:
```suggestion
mix.js('resources/js/create_subject.js', 'public/js/create_subject.js')
```

**Comment by thaonp104** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**